src/test/java/io/kestra/plugin/flink/CancelTest.java

```diff
@@ -0,0 +1,63 @@
+package io.kestra.plugin.flink;
+
+import io.kestra.core.models.property.Property;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+@MicronautTest
+class CancelTest {
```

Comment:
All those tests are `@KestraTest` not `@MicronautTest` and you should also use a `RunContextFactory` to use to actually run the tasks